### DISCUSSION TITLE
nfpcapd does not support cross compiles

### DISCUSSION THREAD

**Discussion by 1337c0d3d00d** [phaag/nfdump#466] [nfdump-1.7]
when enabling --enable-nfpcapd and trying to cross compile for arm with buildroot configure complains and fails here:

"configure: error: cannot run test program while cross compiling"

did not want to hack up configure.ac to get it working.  any assistance would be great.

thank you.

**Comment by phaag** [comment]
That's a bit difficult, as configure or `autotools` does run some test programs which can not be executed, if you cross compile.
This is not really an nfdump issue, but an `autotools` issue.
